primitives/millau/src/lib.rs

```diff
@@ -21,11 +21,17 @@
 #![allow(clippy::unnecessary_mut_passed)]
 
 use bp_runtime::Chain;
-use frame_support::RuntimeDebug;
+use frame_support::{weights::Weight, RuntimeDebug};
 use sp_core::Hasher as HasherT;
-use sp_runtime::traits::BlakeTwo256;
+use sp_runtime::{
+	traits::{BlakeTwo256, IdentifyAccount, Verify},
+	MultiSignature, MultiSigner,
+};
 use sp_std::prelude::*;
 
+/// Maximal weight of single Millau extrinsic.
+pub const MAXIMUM_EXTRINSIC_WEIGHT: Weight = 1_300_000_000_000;
```

Comment:
How did you get this number? It should have some documentation around the process

Reply:
I just took actual value from `Millau` runtime and moved it here along with its comments (none).

Reply:
I've actually replaced computation formula with constant, which may be treated as comment. So I've added some more constants to primitives.

Reply:
Having a formula for `MAXIMUM_EXTRINSIC_WEIGHT` is much better, thanks! Now the question becomes how did we choose `MAXIMUM_BLOCK_WEIGHT`? It looks like it's '2 * WEIGHT_PER_SECOND` which is from Substrate and translates to ~2 seconds of compute. Is that right?

Reply:
Yes, I think you're right - it is roughly 2 sec, iiuc